pkg/cliplugins/claims/cmd/cmd.go

```diff
@@ -85,5 +91,35 @@ func New(streams genericclioptions.IOStreams) *cobra.Command {
 	apibindingGetOpts.BindFlags(apibindingGetCmd)
 	getcmd.AddCommand(apibindingGetCmd)
 	claimsCmd.AddCommand(getcmd)
+
+	editcmd := &cobra.Command{
+		Use:              "edit",
+		Short:            "Operations related to editing the status of permission claims",
+		SilenceUsage:     true,
+		TraverseChildren: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+
+	apibindingEditOpts := plugin.NewEditAPIBindingOptions(streams)
+	apibindingEditCmd := &cobra.Command{
+		Use:          "apibinding <apibinding_name>",
+		Short:        "Edit claims related to apibinding",
```

Comment:
Add doc to say this does all of them, and when

Reply:
Have modified this to be `Edit open permission claims`, since listing all of them is just a one off case which is explained previously in the command help.